docs/operator_subscriptions/transmogrifier.md

```diff
@@ -0,0 +1,121 @@
+| `exclude_site`    | Select an RSE that is **not** the site used by the parent rule (using the `site` attribute to exclude). | Ensuring that different copies are placed at physically separate sites.                 |
+
+- Both algorithms are only valid for rules with split logic and a single copy.
+- Errors are raised if required parameters are missing or misconfigured.
+
+---
+
+## Flowchart: Transmogrifier Behaviour
+
+```mermaid
+flowchart TD
+    A["Start transmogrifier daemon"] --> B["Initialize threads & heartbeat"]
+    B --> C["Load active subscriptions"]
+    C --> D1["Fetch new DIDs"]
+    D1 --> E1{"For each new DID"} & W1{"No more new DIDs"}
+    E1 --> F1["Fetch metadata for DID"]
+    F1 --> G1{"For each subscription"}
+    G1 --> H1{"Does DID match subscription filter?"} & U1{"All subscriptions checked"}
+    H1 -- No --> G1
+    H1 -- Yes --> I1["Parse subscription rules"]
+    I1 --> J1{"For each rule in subscription"}
+    J1 --> K1["Prepare rule parameters<br> :  RSE expression, copies, etc."] & T1{"All rules done for this subscription"}
+    K1 --> L1{"Is rule split/chained?"}
+    L1 -- Yes --> M1["Select RSEs using split/chained logic"]
+    L1 -- No --> N1["Use RSEs from rule directly"]
+    M1 --> O1["For each selected RSE"]
+    N1 --> O1
+    O1 --> P1{"Is RSE blocklisted and rule wildcarded?"}
+    P1 -- Yes --> Q1["Check ignore_availability<br>Skip or create rule"]
+    P1 -- No --> R1["Create replication rule for DID"]
+    Q1 --> R1
+    R1 --> S1["Record created rule and success"]
+    S1 --> J1
+    T1 --> G1
+    U1 --> V1["If successful, mark DID as processed"]
+    V1 --> D1
+    W1 --> X1["Update last_processed in subscriptions"]
+    X1 --> Y1["Push metrics and logs"]
+    Y1 --> Z1["Sleep if not once and repeat or exit"]
+```
+
+---
+
+## Key Concepts
+
+- **DID**: Data Identifier (can be a file, dataset, or container).
+- **Subscription**: User-defined pattern and rules for automatic data placement.
+- **Replication Rule**: Instruction for Rucio to place a copy of a DID on specified RSEs.
+- **RSE**: Rucio Storage Element (storage endpoint, e.g., a data center or cloud bucket).
+- **Split/Chained Logic**: Advanced methods for spreading or chaining rules across multiple sites, using algorithms.
+
+---
+
+## Summary
+
+The transmogrifier daemon is central to Rucio’s automated data management, ensuring that new data is promptly and correctly distributed according to organizational policies and user-defined subscriptions. Its sophisticated logic—including split, chained, and algorithmic rule selection—supports even the most advanced data placement strategies, while robust error handling and metrics allow for reliable, scalable operation.
+
+---
```

Comment:
I'm still in favour of removing this as I don't think it adds anything that's not already in the Purpose section above, but I can understand that this might be a subjective opinion. Maybe we can get someone else to review this as well to give their opinion